Android/Android-Unreal-Plugin/build.gradle.kts

```diff
@@ -2,24 +2,14 @@ plugins {
     id("com.android.library")
 }
 
-private val versionMajor = 1
-private val versionMinor = 0
-private val versionPatch = 0
-
```

Comment:
How do we version the library now?

Reply:
Discussed in meatspace. TL;DR is that we don't version this library and the larger Unreal stuff is unaffected.